src/components/invoices/InvoiceToolBar.js

```diff
@@ -284,6 +261,28 @@ class InvoiceToolBar extends Component {
             </Button>
           </DialogActions>
         </Dialog>
+        <Dialog
+          open={this.state.locked}
+          onClose={this.handleUnLock}
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
+        >
+          <DialogTitle id="alert-dialog-title">{actionState}</DialogTitle>
+          <DialogContent>
+            <DialogContentText id="alert-dialog-description">
+              This will {actionState} Data for the Zone {this.props.invoice.orgUnit && this.props.invoice.orgUnit.name}{" "}
```

Comment:
may be say parent and its orgunits (not always called a zone)
Ideally I would have displayed the don't know how hard it is to turn the dataApproval.orgUnit into a list of orgunits.
I'll try to test that this afternoon.

Reply:
![Capture d’écran de 2021-03-12 11-18-09](https://user-images.githubusercontent.com/6383219/110919650-2c962c80-8325-11eb-845d-715c28a2ee34.png)